sdk/eventhub/Azure.Messaging.EventHubs/src/Core/BasicRetryPolicy.cs

```diff
@@ -137,7 +137,6 @@ private static bool ShouldRetryException(Exception exception)
                     return ex.IsTransient;
 
                 case TimeoutException _:
```

Comment:
This was a copy/paste mistake.  The Event Hubs documentation explicitly calls out that they use `OperationCanceledException` in scenarios where they do not want you to retry.